src/main/java/org/gridsuite/study/server/dto/ShortCircuitPredefinedConfiguration.java

```diff
@@ -11,5 +11,6 @@
  */
 public enum ShortCircuitPredefinedConfiguration {
     ICC_MAX_WITH_NOMINAL_VOLTAGE_MAP,
-    ICC_MAX_WITH_CEI909
+    ICC_MAX_WITH_CEI909,
+    ICC_MIN_WITH_NOMINAL
```

Comment:
I think you should change this. **ICC_MIN_WITH_NOMINAL_VOLTAGE_MAP** is more  convenient.